commit dde9f9f7e404b82aaf7f86c5912e7d27481bdc8e
Author: Panu Matilainen <pmatilai@redhat.com>
Date:   Wed Mar 4 08:07:50 2015 +0200

    ethdev: add missing symbol export for rte_eth_dev_release_port
    
    Fixes: 36ec8585b298 ("ethdev: release port")
    
    Signed-off-by: Panu Matilainen <pmatilai@redhat.com>

diff --git a/lib/librte_ether/rte_ether_version.map b/lib/librte_ether/rte_ether_version.map
index 0d46578..a2d25a6 100644
--- a/lib/librte_ether/rte_ether_version.map
+++ b/lib/librte_ether/rte_ether_version.map
@@ -43,6 +43,7 @@ DPDK_2.0 {
 	rte_eth_dev_mac_addr_add;
 	rte_eth_dev_mac_addr_remove;
 	rte_eth_dev_priority_flow_ctrl_set;
+	rte_eth_dev_release_port;
 	rte_eth_dev_rss_hash_conf_get;
 	rte_eth_dev_rss_hash_update;
 	rte_eth_dev_rss_reta_query;
